src/Symfony/Component/DependencyInjection/Tests/EnvVarProcessorTest.php

```diff
@@ -997,16 +997,7 @@ public static function provideGetEnvUrlPath()
     }
 
     /**
-     * @testWith ["http://foo.com\\bar"]
-     *           ["\\\\foo.com/bar"]
-     *           ["a\rb"]
-     *           ["a\nb"]
-     *           ["a\tb"]
-     *           ["\u0000foo"]
-     *           ["foo\u0000"]
-     *           [" foo"]
-     *           ["foo "]
-     *           [":"]
+     * @dataProvider provideEnvBadUrl
```

Comment:
I suppose can json-encode the \v instead?

Reply:
Indeed!